### PR TITLE
[ZEVMITSD-119] Setup base Cloud Run runtime for Administration portal

### PR DESCRIPTION
**Pull request by bartosz347**
- Work in progress – load balancer is not yet configured
- Created basic config to run the Administration Portal in Cloud Run

### CODE REVIEW

**Comment by cruddasj** [comment]
@bartosz347 - FYI - I've added a source commit hash variable (which is also used when determining image to deploy). This is a particularly helpful pattern (and feel free to use elsewhere!) that does 2 things...

First is you can tell exactly what code is running at any time in any environment (as the commit hash is visible in website layers through a build controller). An example can be seen at https://apply.informeddecision.io/build-id.

Second, is that when we have tests in CI pipelines (e.g., ZAP, Pa11y etc.), which should only run one blue/green rollover has finished - there's a bash script at https://github.com/InformedSolutions/informed-decision-iac/blob/develop/ci-scripts/check_build_id.sh which will await the current build ID to come back. This avoids needing to do anything weird like random waits in the pipeline and you know the code being tested is a 100% match to what has just been deployed 👍 

When deploying from a local environment, you can just set this to latest (if you want to) but you can also roll back to specific images so it's a simple solution that solves many problems at once...

File: src/services/websites/zev-administration-portal/Makefile [hunk 1]
```diff
@@ -13,11 +13,17 @@ clean-build:
 docker-build:
 	cd ../../../.. && docker build -f DfT.ZEV.Administration.Dockerfile -t zev-administration .
 
+docker-push:
+	docker tag zev-administration ${REGISTRY_URL}/zev-administration-portal:${SOURCE_COMMIT_HASH}
+	docker push ${REGISTRY_URL}/zev-administration-portal:${SOURCE_COMMIT_HASH}
```
Comment (bartosz347) [inline src/services/websites/zev-administration-portal/Makefile:18]: @cruddasj  Am I correct that the code below should be removed?
```
	docker tag zev-administration ${REGISTRY_URL}/zev-administration-portal:latest
	docker push ${REGISTRY_URL}/zev-administration-portal:latest
```

Comment (cruddasj) [reply src/services/websites/zev-administration-portal/Makefile:18]: No - you can keep that in (it should just tag both) 👍 So we'll have "latest" which is the latest, and the specific commit hash

**Review by cruddasj — APPROVED** [review]

File: src/services/websites/zev-administration-portal/Makefile (hunk 1 above)
Comment (bartosz347) [reply src/services/websites/zev-administration-portal/Makefile:18]: Ok, reverted

**Pull request merged**